guides/common/modules/proc_building-a-custom-discovery-image.adoc

```diff
@@ -0,0 +1,98 @@
+[id="Building_a_custom_Discovery_Image_{context}"]
+= Building a custom Discovery image
+
+You can build a custom Discovery ISO or rebuild an ISO if you change configuration files.
+
+The Discovery image uses a minimal operating system that is booted on hosts to acquire initial hardware information and check in with {Project}.
+Discovered hosts keep running the Discovery operating system until they are rebooted into an operating system installer, which then initiates the provisioning process.
+
+[IMPORTANT]
+====
+Do not use your production {ProjectServer} nor {SmartProxyServer} to perform this procedure.
+Use either a dedicated environment or copy the repositories and Kickstart file to a separate server.
+====
+
+The following procedure demonstrates the building process on {EL} 8 and uses CentOS Stream 8 repositories.
```

Comment:
CentOS Stream 8 is EoL: https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/

Reply:
TBH I would prefer to drop this procedure altogether. I don't see why anybody would need it when the built FDI is already provided by Foreman.

Reply:
Don't know, will ask internally. Either we drop the procedure in a follow-up PR or we need to raise an issue for FDI.